projects/storefrontlib/cms-components/misc/icon/icon-loader.service.ts

```diff
@@ -21,11 +26,17 @@ import {
 })
 export class IconLoaderService {
   private loadedResources: string[] = [];
+
+  protected renderer: Renderer2;
+
   constructor(
     protected winRef: WindowRef,
     protected iconConfig: IconConfig,
-    protected sanitizer: DomSanitizer
-  ) {}
+    protected sanitizer: DomSanitizer,
+    rendererFactory: RendererFactory2
```

Comment:
To add a constructor arg we need to make sure this code goes in 6.0.

Reply:
Ticket was not approved for 6.0 so I have removed the breaking changes to this file.  

I am keeping the unit test changes since they are a non-breaking and reasonable alternative to checking the usage of `createElement()` and will still meet the goal of removing the deprecated usage.